integration_tests/test_basic.py

```diff
@@ -677,9 +680,16 @@ def test_tx_inclusion(cronos, max_gas_wanted):
 
     test against different max-gas-wanted configuration.
     """
+
+    def fn(cmd):
+        global origin_cmd
+        if origin_cmd is None:
+            origin_cmd = cmd
+        return f"{origin_cmd} --evm.max-tx-gas-wanted {max_gas_wanted}"
+
     modify_command_in_supervisor_config(
         cronos.base_dir / "tasks.ini",
-        lambda cmd: f"{cmd} --evm.max-tx-gas-wanted {max_gas_wanted}",
+        lambda cmd: fn(cmd),
```

Comment:
What problem does it solve?
Do you mean when the test case run multiple times, the flags get appended multiple times?

Reply:
[yes](https://github.com/crypto-org-chain/cronos/pull/901#discussion_r1125779249)